src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --blue-color: #00A7E1;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use the `*` selector as it impacts performance. Set styles only for elements that require them. It's better to define specific styles for specific elements or use classes.